src/test/java/usecases/RunCourseUseCase/RunCourseInteractorTest.java

```diff
@@ -0,0 +1,86 @@
+package usecases.RunCourseUseCase;
+import frameworksdriversmock.DatabaseDriverMock;
+import interfaceadapters.RunCourseInterfaceAdapters.RunCoursePresenter;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Driver;
+
+public class RunCourseInteractorTest {
```

Comment:
Great test coverage for your interactor! All you need to do is add documentation. If you want, you can make "questions" and "answers" instance variables, so that you don't have to initialize them every time, and your methods would be shorter and without duplicates.